src/components/payment/staff-payment.vue

```diff
@@ -1,8 +1,8 @@
 <template>
-  <div id="staff-payment" class="pt-6">
-    <staff-payment-component
+  <div id="staff-payment" :class="{'invalid-section': showStaffPaymentInvalidSection}">
```

Comment:
To keep the component styling simple, I set this class here, in its parent. (This is also what is done in Edit UI for this component.)